man/runc-kill.8.md

```diff
@@ -7,9 +7,48 @@
 Where "<container-id>" is the name for the instance of the container and
 "<signal>" is the signal to be sent to the init process.
 
+The mapping between "<signal>" and the real golang signal is:
+
+    "ABRT":   syscall.SIGABRT,
```

Comment:
The values have to be related to `signal(7)`, because that defines the values that the language uses (it's written by the authors of the kernel). As for the mappings, I'm fairly sure we follow the same convention as `kill(1)` and I don't like copy-pasting Go code into our man pages.